eng/lang.ini

```
@@ -379,3 +379,6 @@ nukkit.resources.success=Successfully loaded {%0} resource packs
 nukkit.resources.zip.not-found=File "{%0}" is not found
 nukkit.resources.zip.no-manifest="manifest.json" is not found
 nukkit.resources.zip.invalid-manifest="manifest.json" is invalid or incomplete
+nukkit.resources.plugin.loaded= Successfully loaded resource pack from plugin {%0}: {%1}
+nukkit.resources.plugin.loading= Loading resource pack from plugin {%0}
+nukkit.resources.zip.loaded= Resource pack loaded successfully: {%0}
```

Remove the extra space before each value, it's the same for the rest of the files.
```suggestion
nukkit.resources.plugin.loaded=Successfully loaded resource pack from plugin {%0}: {%1}
nukkit.resources.plugin.loading=Loading resource pack from plugin {%0}
nukkit.resources.zip.loaded=Resource pack loaded successfully: {%0}
```